test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -379,6 +379,9 @@ class CloudFoundryDeployTest extends BasePiperTest {
         assertThat(shellRule.shell, hasItem(containsString('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"')))
         assertThat(shellRule.shell, hasItem(containsString("cf blue-green-deploy testAppName -f 'test.yml'")))
         assertThat(shellRule.shell, hasItem(containsString("cf stop testAppName-old &> 1-cfStopOutput.txt")))
+
+        UUID.metaData = null
```

Comment:
In case we exit above in one or the asserts this coding here is not executed. That means failing in the asserts affects other test. We can say: "well, in case a test fails everything else is undefinded anyhow" or can

1.) put it into a finally
2.) put it into a teardown method (annotated with `@After`, maybe `@AfterClass` is also possible)